ConfigSpace/api/types/float.py

```diff
@@ -153,7 +155,8 @@ def Float(
             log=log,
             meta=meta,
         )
-    elif isinstance(distribution, Normal):
+
+    if isinstance(distribution, Normal):
```

Comment:
Is this considered more readable?

Reply:
It's certainly a possible point of debate, I just went with a default of `ruff` which say it's unnecessary. There are cases where it is more readable in my opinion but here I have no real strong say.